### PR TITLE
Update Arcade to v1.16

### PR DESCRIPTION
**Pull request by jf990**
Update Arcade grammar to ArcGIS Arcade version 1.16: new keywords, code clean up, relevance update, v11 update.

### Changes
languages arcade.js updated

### Checklist
- [x] Verified markup tests
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

File: src/languages/arcade.js [hunk 1]
```diff
@@ -75,7 +249,8 @@ export default function(hljs) {
   ]);
 
   return {
-    name: 'ArcGIS Arcade',
+    aliases: [ 'arcade' ],
```
Comment (joshgoebel) [inline src/languages/arcade.js:252]: Where did name go?  `alias` isn't necessary if it just matches the filename.

File: src/languages/arcade.js [hunk 2]
```diff
@@ -75,7 +249,8 @@ export default function(hljs) {
...
+    case_insensitive: true,
```
Comment (joshgoebel) [inline src/languages/arcade.js:253]: This quite possibly could be why your relevance broke (if this is the language you were referring to)... that changes a lot of things (makes them less precise, more likely to false positive, etc).

Comment (joshgoebel) [inline src/languages/arcade.js:253]: The big changes here were keywords and case insensitive, right?    That and relevance?  If I back out all the tiny changes (that I asked you to change back) and just copied over the keywords and case sensitivity and took a look at the relevance issues myself, would that achieve the goals here?

I know you wanted to pull this out so you were probably tuning this a bit more to your liking, but now that it's staying...

Comment (jf990) [reply src/languages/arcade.js:253]: @joshgoebel that's correct the only real changes are the addition of new keywords in the recent language update and case insensitive. The relevance was changed only to get the tests to pass so it has no functional significance to the parsing. So I'm fine with working out what you think is best so we can get this in the next update.

File: src/languages/arcade.js (hunk 1 above)
Comment (jf990) [reply src/languages/arcade.js:252]: not sure, i think this was unintentional. it should be `arcade`, I'll revert `name`.

File: src/languages/arcade.js (hunk 2 above)
Comment (jf990) [reply src/languages/arcade.js:253]: @joshgoebel I just restored `name`

not sure what you want to do about `relevance`, do you want me to revert all of those changes?

Comment (joshgoebel) [reply src/languages/arcade.js:253]: You don't need to do anything further, I'll take a pass from scratch and tag you on the keyword/sesnativity change.

**Pull request closed without merging**